### ISSUE TITLE
fast_anchor are not SEO crawlable

### ISSUE THREAD

**Issue by ameyasubhedar** [microsoft/fast#5951]
I noticed that the fast_anchor and FluentAnchor generate `<a>` elements that are not SEO optimized.

Running the Lighthose on the web app that have fast_anchor shows the report below, none of the anchor could be crawled.

![image](https://user-images.githubusercontent.com/13166236/167197609-5e21abb1-ab8f-4acd-8962-ed09298229fc.png)

As a results the links are not shown in the search results, this needs to be fixed.

**Comment by chrisdholt** [comment]
I'm not quite sure what we can do here as we're using the platform to delegate focus to a semantic element here. The alternative would be to make all the actual actionable elements themselves slotted. Open to suggestions here, but I'm not sure we have a good alternative here with delegates focus.

**Comment by ameyasubhedar** [comment]
@chrisdholt , @EisenbergEffect . I did some more digging around and initially I thought this issue is caused by the fast_anchor but apparently this looks like an issue with Blazor Webassembly default error handler that adds 2 `<a>` elements for reloading and dismissing the the error message. I raised an issue with Blazor team https://github.com/dotnet/aspnetcore/issues/41600.

**Comment by EisenbergEffect** [comment]
Thanks for following back up. We'll close this issue then and let Blazor pick it up on their side. If something else comes up we can open a new issue as needed.
